### ISSUE TITLE
Model ends up without texture

### ISSUE THREAD

**Issue by sayduck-daniel** [kcoley/gltf2usd#68]
I'm trying to get a GLTF converted in USDZ but somehow, my model ends up without any texture. When I uncompress the .usdz the texture file is in there.

For some context, I installed last week USD by Pixar and gltf2usd on a Debian on AWS EC2. It worked, I was able to export and get a nice looking USDZ.

Then I proceed to do the same again but on an Amazon Linux 2 on AWS EC2 and after some first tests, I see the USDZ doesn't display its texture. I taught it was because of some library compatibility or such kind of problem so I keep digging but now when I resolve myself to go back to use Debian I try again to convert the same GLTF and I get the same without texture model.

I wonder if this could come grom gltf2usd? Is there something obvious I'm missing here?

**Comment by sayduck-daniel** [comment]
- This zip can be extracted and contains the GLTF files: [example_multi_meshes.zip](https://github.com/kcoley/gltf2usd/files/2453858/example_multi_meshes.zip)

- This one is the USDZ file renamed, just rename it back to .usdz: [example_multi_meshes-01.usdz.zip](https://github.com/kcoley/gltf2usd/files/2453859/example_multi_meshes-01.usdz.zip)

**Comment by kcoley** [comment]
Hi @sayduck-daniel .  This is definitely odd.  I am also seeing all files and references working when opening the usdz file.  I also locally converted the original glTF and it looks to work as expected.  Are you converting to usdz with `usdzip` or are you using `gltf2usd` to do this directly?  

Also are you on the lastest master of `gltf2usd` and on Pixar's `USD`?

**Comment by sayduck-daniel** [comment]
ok... I understand what is the problem...

It was no need for me to build 30h of source code for libraries needed by USD or for me to have to compile from source LLVM or CLANG.

The problem comes from how I call gltf2usd.

Initially, I was doing it with a full path:

    python /usr/local/gltf2usd/Source/gltf2usd.py -g ~/workspace/variant.gltf -o ~/workspace/variant.usdz

Then when I switch from Debian to Amazon Linux 2 on EC2, while installing stuff I simplify my script to do instead:

    cd  ~/workspace/
    python /usr/local/gltf2usd/Source/gltf2usd.py -g ./variant.gltf -o ./variant.usdz

So long story short, I just spent 80h in total for nothing 👍

Relative paths, is it something on your side or USD side? I can open an issue on their repo if needed.

**Comment by kcoley** [comment]
Ah that makes sense. I think the relative paths would probably have to be handled within this tool. Feel free to file and issue here or make a PR if you think it’s a quick fix!

**Comment by sayduck-daniel** [comment]
Could you add this to the README file? Just in case someone else false in this trap.

**Comment by kcoley** [comment]
@sayduck-daniel actually I had a thought. The converter was able to transfer the textures so the issue may actually be with usdzip. I’ll try to reproduce when I get some time. But I’ll make a note to update the readme

**Comment by kcoley** [comment]
@sayduck-daniel I just merged a change to support relative paths in the generated usdz file.  I basically change to the directory of the usdc file and switch all paths to relative so `CreateNewUsdzPackage` works as expected.  Feel free to give this a try.

**Comment by sayduck-daniel** [comment]
Thanks, I'm gonna try o^when we next deploy the latest of USD and your code. I let you know ASAP

**Comment by MrDChristop** [comment]
Usdzip can not handle relative paths, i already filed a report on USD forum
https://groups.google.com/forum/#!topic/usd-interest/jos8g0t8w00

I am not sure if gltf2usd should support this hackfix.
I also do this manually in my code, change to the directory of where the usd has to be created.

**Comment by MrDChristop** [comment]
As foretold this line which used to work now fails
gltf2usd.py -g sdf_vray_utf8.gltf -o das.usdz
```
Traceback (most recent call last):
  File "C:\Users\d_chr\Downloads\gltf2usd-master\Source\gltf2usd.py", line 742, in <module>
    convert_to_usd(os.path.expanduser(args.gltf_file), os.path.expanduser(args.usd_file), args.fps, args.scale, args.arkit, args.verbose, args.use_euler_rotation)
  File "C:\Users\d_chr\Downloads\gltf2usd-master\Source\gltf2usd.py", line 707, in convert_to_usd
    os.chdir(os.path.dirname(usdc_file))
WindowsError: [Error 123] The filename, directory name, or volume label syntax is incorrect: ''
```

This line has to be given now in the latest version instead
gltf2usd.py -g sdf_vray_utf8.gltf -o ./das.usdz

So you MUST specify a directory now.... We exchanged one evil with another..

**Comment by kcoley** [comment]
@MrDChristop thanks for flagging the issue.  I made a recent change that converts the usd file path to absolute path for now.

**Comment by MrDChristop** [comment]
Hi. Where is the commit for this?

**Comment by kcoley** [comment]
@MrDChristop the commit is here at the bottom of gltf2usd.py: https://github.com/kcoley/gltf2usd/pull/83/files
